components.yaml

```diff
@@ -13,10 +13,10 @@ components:
     metadata: v0.0.15
   kubemacpool:
     url: https://github.com/k8snetworkplumbingwg/kubemacpool
-    commit: 22482e75a4fb4ea986d03f7876bc0f1241860a0e
+    commit: b94b42a8d249cdefa9bf1473c4f84fa443822b5e
     branch: release-0.44
     update-policy: latest
-    metadata: v0.44.1
+    metadata: v0.44.1-2-gb94b42a
```

Comment:
what is the `2-g` that is added ? it is expected?

Reply:
yes. it means 2 commit behind the tag it originated from (`v0.44.1`).
It's OK.

Reply:
regarding the `g` - not sure what it stands for.. checking

Reply:
ahead, not behind right ?

Reply:
yes, sorry.